.github/workflows/nodepackage.yml

```diff
@@ -46,3 +46,10 @@ jobs:
           fi
           npm install
           npm test
+      - name: Lint
+        run: |
+          if [ ${{ matrix.node }} = 6 ]; then
```

Comment:
I guess lower versions are not working maybe use "-le" instead of "="

Reply:
The engines stanza in package.json already says the package only supports Node 6+.